lib/src/cmaf/ham/Track.ts

```diff
@@ -1,3 +1,5 @@
+import { VideoTrack } from "./VideoTrack";

```

Comment:
Doesn't this one need the `.js` at the end?

Reply:
Yes, it is needed. Resolved.